doc/source/getting_started/write_documentation.rst

```diff
@@ -0,0 +1,224 @@
+.. _write_documentation:
+
+===================
+Write documentation
+===================
+
+Writing documentation is an excellent way to contribute to a project because it
+plays a pivotal role in making the project more accessible and usable. Clear and
+comprehensive documentation empowers users and developers to understand,
+implement, and troubleshoot the project effectively. It minimizes barriers to
+entry, making it easier for newcomers to get involved and for existing
+contributors to be more productive.
+Good documentation also reduces the burden on maintainers,
+as it can answer common questions and help prevent issues. By
+creating or improving documentation, you not only enhance the project's quality
+but also facilitate knowledge sharing and community growth, making your
+contribution invaluable for the project's long-term success.
+
+Setting up your environment
+===========================
+
+To be able to write and build the documentation, you need to follow the same
+steps as :ref:`developing_pymapdl` but this case make sure you install
+the documentation dependencies:
+
+.. code:: console
+
+    pip install -e '.[doc]'
+
+
+Building the documentation
+==========================
+
+PyMAPDL documentation is mainly written in reStructuredText
+format, saved as ``.rst`` files in the ``doc/source`` directory.
+The tool used to  build the documentation from these reStructuredText files 
+is `Sphinx <sphinx_>`_.
+Sphinx also build API documentation from source code as well as manager the
+cross-referencing between the different files, classes, methods, etc.
+Additionally, it also build a `examples gallery <pymapdl_examples_gallery_>`_
+where the capabilities of PyMAPDL can be showcased.
+The documentation can be built as HTML files or a single PDF file.
+
+To build the documentation as HTML files you just need to run:
+
+.. code:: console
+
+   make -C doc html
+
+On Windows: 
+
+.. code:: pwsh-session
+
+    doc\make.bat html
+
+This command build the documentation as HTML in the directory ``doc/_build/html``.
+
+If you want to build the PDF documentation, you need first to install
+a latex distribution as `MikTeX <miktex_>`_. Then run:
+
+.. code:: console
+
+   make -C doc pdf
+
+Both commands, run the Python files in ``./examples`` in the repository root directory
+to generate the `examples gallery <pymapdl_examples_gallery_>`_.
+The result of running these examples is cached, so the next time only the changed files
+are re-run.
```

Comment:
```suggestion
Running the command to build either HTML files or a PDF file runs the Python files in ``./examples`` in the repository root directory to generate the `examples gallery <pymapdl_examples_gallery_>`_.
The result of running these examples is cached so that the only the changed files
are re-run the next time.
```